addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -16,7 +16,8 @@ class StockPickingBatch(models.Model):
 
     name = fields.Char(
         string='Batch Transfer', default='New',
-        copy=False, required=True, readonly=True)
+        copy=False, required=True)
+    description = fields.Char(size=35)
```

Comment:
Better add a string just to be safe
```suggestion
    description = fields.Char("Description", size=35)
```